tests/h/views/api_groups_test.py

```diff
@@ -143,19 +205,24 @@ def username_request(self, pyramid_request):
         pyramid_request.matchdict['user'] = 'bob'
         return pyramid_request
 
-    @pytest.fixture
-    def group_service(self, pyramid_config):
-        service = mock.create_autospec(GroupService, spec_set=True, instance=True)
-        pyramid_config.register_service(service, name='group')
-        return service
-
     @pytest.fixture
     def authenticated_userid(self, pyramid_config):
         userid = 'acct:bob@example.org'
         pyramid_config.testing_securitypolicy(userid)
         return userid
 
 
+@pytest.fixture
+def anonymous_request(pyramid_request):
+    pyramid_request.user = None
+    return pyramid_request
```

Comment:
We should probably move this to `tests/h/conftest.py` at some point as it seems pretty generally useful.